src/main/resources/alma/maps/classificationCode2Label.tsv

```diff
@@ -0,0 +1,21 @@
+udc	UDC (Universal Decimal Classification)
+sdnb	DNB (Systematik der Deutschen Nationalbibliographie)
+methepp	Methode Eppelsheimer
+bkl	BK (Basisklassifikation)
+rvk	RVK (Regensburger Verbundklassifikation)
+ghbs	Gesamthochschulbibliothekssystematik (GHBS)
+njb	NDC (Nippon Decimal Classification)
+kktb	NDLC (National Diet Library Classification)
+rpb	RPB (Rheinland-Pfälzische Bibliographie)
+msc	MSC (Mathematics Subject Classification)
+nwbib	NWB (Nordrhein-Westfälische Bibliographie)
```

Comment:
As NWbib is not in classificationLabel2Uri.tsv shouldn't we also remove it here?